lib/maths/CTimeSeriesModel.cc

```diff
@@ -573,11 +598,13 @@ bool CTimeSeriesAnomalyModel::acceptRestoreTraverser(const SModelRestoreParams&
                         &m_AnomalyFeatureModels[index++], _1)))
         }
     }
+    // else we can't upgrade the state of the anomaly model pre 6.5.
+
     return true;
 }
 
 void CTimeSeriesAnomalyModel::acceptPersistInserter(core::CStatePersistInserter& inserter) const {
-    inserter.insertValue(VERSION_6_5_TAG, "");
+    inserter.insertValue(VERSION_7_2_TAG, "");
     if (m_Anomaly) {
         inserter.insertLevel(ANOMALY_6_5_TAG, boost::bind(&CAnomaly::acceptPersistInserter,
```

Comment:
This is no longer restored, so is it worth persisting it?  Or is it a mistake not to restore it?

Reply:
I'd deleted the restore code from the wrong branch. Good catch.